# Automatically Generated in 2017-06-12 22:33:47.453077
# MESSAGE: CONFIG_REQUEST_PARAMS
# Description:Request config parameter information
uint8 ID = 194
uint8 sysid
uint8 compid
uint8 config_id
uint8 param_id
